src/fast++-read_input.cpp

```diff
@@ -1656,6 +1663,11 @@ bool check_input(options_t& opts, input_state_t& state) {
         }
     }
 
+    // If no photometry is present then disable the auto-scaling of the spectrum
+    if (state.flux.dims[1] == 0) {
+      opts.auto_scale = false;
```

Comment:
```suggestion
        if (opts.auto_scale) {
            warning("AUTO_SCALE was disabled because no photometry was provided");
            opts.auto_scale  = false;
        }
```

Reply:
Just letting the user know what's happening under the hood, in case they made a mistake by not providing any photometry.